async-http-client-backend/zio1/src/test/scala/sttp/client3/asynchttpclient/zio/SttpBackendStubZioTests.scala

```diff
@@ -73,4 +74,45 @@ class SttpBackendStubZioTests extends AnyFlatSpec with Matchers with ScalaFuture
 
     runtime.unsafeRun(effect).map(_.body).toList shouldBe List(Right("a"), Right("b"), Right("c"), Right("a"))
   }
+
+  it should "lift errors due to mapping with impure functions into the response monad" in {
```

Comment:
I think that we have test that alredy cover this case in HttpTest. Are they not working ?
What is the difference between them and this new tests - besides using exact monad not generic one ?

Reply:
These new tests validate `SttpBackendStub`. HttpTests are not run against the stub backend which has its own set of tests.

Reply:
Ok so just this comment up